src/main/java/kr/sesac/aoao/server/diary/controller/dto/request/DiaryCreateRequest.java

```diff
@@ -0,0 +1,14 @@
+package kr.sesac.aoao.server.diary.controller.dto.request;
+
+import java.time.LocalDateTime;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public class DiaryCreateRequest {
+	private final Long userId;
+	private final LocalDateTime date;
+	private final String content;
+}
```

Comment:
```suggestion
@Getter
@RequiredArgsConstructor
public class DiaryCreateRequest {
	private final Long userId;
	private final LocalDateTime date;
	private final String content;
}
```
`@RequiredArgsConstructor` 는 final 이 붙은 필드를 포함하는 생성자를 만들어줍니다 ㅎㅎ